draft-ietf-oauth-identity-chaining.md

```diff
@@ -198,13 +198,13 @@ Content-Type: application/json
 Cache-Control: no-cache, no-store
 
 {
-  "access_token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJo
+  "access_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJo
   dHRwczovL2FzLmEub3JnL2F1dGgiLCJleHAiOjE2OTUyODQwOTIsImlhdCI6MTY5N
   TI4NzY5Miwic3ViIjoiam9obl9kb2VAYS5vcmciLCJhdWQiOiJodHRwczovL2FzLm
   Iub3JnL2F1dGgifQ.304Pv9e6PnzcQPzz14z-k2ZyZvDtP5WIRkYPScwdHW4",
-  "token_type":"N_A",
+  "token_type": "N_A",
   "issued_token_type": "urn:ietf:params:oauth:token-type:jwt",
```

Comment:
Note that this already had a space. I assume we should have spaces, but if not, then we can do it the other way around. I am not married to either style.

Reply:
Adding a space on the two lines with `"access_token":` kinda throws off the formatting of the right side of the example. I'd want to avoid that by re-flowing the JWT in them to have uniform line breaks on that side and still stay under the line length (suggested) limit. Or just go with the no spaces style throughout. The latter is probably easier/better anyway. If we even really need to have consistent spacing...

Reply:
yeah, I am ok with no spaces if that is easier and ok.